index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Calculator</title>
+  <link rel="stylesheet" href="css/main.css">
+  <script src="js/screen.js" defer></script>
+  <script src="js/keyboard.js" defer></script>
+  <script src="js/calculator.js" defer></script>
+  <script src="js/app.js" defer></script>
+</head>
+<body>
+<div class="calculator" data-component="calculator">
+  <div class="screen" data-component="screen">
+    <span class="screen-value" data-component="screen-result">0</span>
+  </div>
+  <div class="keyboard" data-component="keyboard">
+    <table>
+      <tr class="first">
+        <td data-selector="key" data-key-value="percent">%</td>
+        <td data-selector="key" data-key-value="sqrt">&#8730</td>
+        <td data-selector="key" data-key-value="sqr">&#119909&#178</td>
+        <td data-selector="key" data-key-value="fraction">&#185/&#119909</td>
+      </tr>
+      <tr>
+        <td class="text" data-key-value="ce">CE</td>
+        <td class="text" data-key-value="c">C</td>
+        <td data-key-value="back-space">⇐</td>
```

Comment:
It's good to use one practice for all application. In this case it would be char code (e.g. &#8730)